task8/{UPMessenger}/src/main/java/GetName.java

```diff
@@ -0,0 +1,23 @@
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+@WebServlet("/get")
+public class GetName extends HttpServlet {
+
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+
+        response.setContentType("text/html");
+        String name = request.getParameter("name");
+        PrintWriter writer = response.getWriter();
+        try {
+            writer.println("<span>Name Is " + name + "</span>");
+        } finally {
+            writer.close();
```

Comment:
Actually, you should not close the writer, leave it for the servlet container.
The most general principle is "You close what you opened. You don't close what you haven't opened"
You haven't opened a writer, just took a reference to it "ready-made"